presto-native-execution/presto_cpp/main/http/HttpServer.cpp

```diff
@@ -75,6 +78,16 @@ void sendErrorResponse(
       .sendWithEOM();
 }
 
+void sendErrorJsonResponse(
```

Comment:
There is both a json and thrift for OkResponse, but the error has only json. Should we consider adding a thrift response as well ?

@tdcmeehan

Reply:
I'm not quite sure what this means, but to be clear, we'll only send JSON back from this endpoint, never Thrift.

Reply:
My comment was a bit cryptic, but yes, I wanted to know if the native side-car will work with both JSON and Thrift.
Prestissimo has support for both formats for the control API. Its likely the thrift usage is legacy. 

But if its not too much work, lets add a Thrift one for consistency.

Reply:
@aditi-pandit , just so I understand what you're asking for a little better - are you asking for an additional API in HttpServer that can send an error formatted as Thrift, e.g. `void sendErrorThriftResponse(proxygen::ResponseHandler* downstream,const std::string& body)` or are you also asking that the `/v1/velox/plan` endpoint could also send responses back in Thrift format? 

If it's the latter, I think there would have to be some additional conversion of the message to Thrift and it doesn't sound like it would be useful since only JSON is being used for this case.

Reply:
@BryanCutler : It is the former. 

It's not worth changing /v1/velox/plan to send responses in Thrift format.